### ISSUE TITLE
osx packaging results in venv error

### ISSUE THREAD

**Issue by lutznico** [kivy/buildozer#345]
HI, 
I am trying to release my project with the help of buildozer. After successfully initialzing it gives me the following output:

 ```
mac:pack nico$ buildozer osx debug
# Check configuration tokens
# Ensure build layout
# Create directory /Users/nico/packer/pack/.buildozer
# Create directory /Users/nico/packer/pack/bin
# Create directory /Users/nico/packer/pack/.buildozer/applibs
# Create directory /Users/nico/packer/pack/.buildozer/osx/platform
# Create directory /Users/nico/packer/pack/.buildozer/osx/app
# Check configuration tokens
# Preparing build
# Check requirements for osx
# Check if kivy-sdk-packager exists
# kivy-sdk-packager does not exist, clone it
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   127    0   127    0     0    162      0 --:--:-- --:--:-- --:--:--   163
100  191k    0  191k    0     0   101k      0 --:--:--  0:00:01 --:--:--  205k
Archive:  master.zip
c013d00bbaca6ff8ef6c806d0f1d148c9c0c1b39
   creating: kivy-sdk-packager-master/
  inflating: kivy-sdk-packager-master/.gitignore  
  inflating: kivy-sdk-packager-master/LICENSE  
  inflating: kivy-sdk-packager-master/README.md  
 extracting: kivy-sdk-packager-master/__init__.py  
  inflating: kivy-sdk-packager-master/appveyor.yml  
   creating: kivy-sdk-packager-master/linux/
   creating: kivy-sdk-packager-master/linux/debian/
  inflating: kivy-sdk-packager-master/linux/debian/README.md  
   creating: kivy-sdk-packager-master/linux/debian/daily/
  inflating: kivy-sdk-packager-master/linux/debian/daily/changelog  
 extracting: kivy-sdk-packager-master/linux/debian/daily/compat  
  inflating: kivy-sdk-packager-master/linux/debian/daily/control  
  inflating: kivy-sdk-packager-master/linux/debian/daily/copyright  
 extracting: kivy-sdk-packager-master/linux/debian/daily/kivy-doc.docs  
 extracting: kivy-sdk-packager-master/linux/debian/daily/kivy-examples.install  
  inflating: kivy-sdk-packager-master/linux/debian/daily/kivy-tools.install  
  inflating: kivy-sdk-packager-master/linux/debian/daily/python-kivy-bin.install  
 extracting: kivy-sdk-packager-master/linux/debian/daily/python-kivy-common.install  
  inflating: kivy-sdk-packager-master/linux/debian/daily/python-kivy.install  
  inflating: kivy-sdk-packager-master/linux/debian/daily/python3-kivy-bin.install  
 extracting: kivy-sdk-packager-master/linux/debian/daily/python3-kivy-common.install  
  inflating: kivy-sdk-packager-master/linux/debian/daily/python3-kivy.install  
  inflating: kivy-sdk-packager-master/linux/debian/daily/rules  
   creating: kivy-sdk-packager-master/linux/debian/stable/
  inflating: kivy-sdk-packager-master/linux/debian/stable/changelog  
 extracting: kivy-sdk-packager-master/linux/debian/stable/compat  
  inflating: kivy-sdk-packager-master/linux/debian/stable/control  
  inflating: kivy-sdk-packager-master/linux/debian/stable/copyright  
 extracting: kivy-sdk-packager-master/linux/debian/stable/kivy-doc.docs  
 extracting: kivy-sdk-packager-master/linux/debian/stable/kivy-examples.install  
  inflating: kivy-sdk-packager-master/linux/debian/stable/kivy-tools.install  
  inflating: kivy-sdk-packager-master/linux/debian/stable/python-kivy-bin.install  
 extracting: kivy-sdk-packager-master/linux/debian/stable/python-kivy-common.install  
  inflating: kivy-sdk-packager-master/linux/debian/stable/python-kivy.install  
  inflating: kivy-sdk-packager-master/linux/debian/stable/python3-kivy-bin.install  
 extracting: kivy-sdk-packager-master/linux/debian/stable/python3-kivy-common.install  
  inflating: kivy-sdk-packager-master/linux/debian/stable/python3-kivy.install  
  inflating: kivy-sdk-packager-master/linux/debian/stable/rules  
   creating: kivy-sdk-packager-master/osx/
  inflating: kivy-sdk-packager-master/osx/README.md  
  inflating: kivy-sdk-packager-master/osx/cleanup_app.sh  
  inflating: kivy-sdk-packager-master/osx/create-osx-bundle.sh  
  inflating: kivy-sdk-packager-master/osx/create-osx-dmg.sh  
   creating: kivy-sdk-packager-master/osx/data/
  inflating: kivy-sdk-packager-master/osx/data/MakeSymlinks  
  inflating: kivy-sdk-packager-master/osx/data/background.png  
  inflating: kivy-sdk-packager-master/osx/data/icon.icns  
  inflating: kivy-sdk-packager-master/osx/data/link_duplicate.py  
  inflating: kivy-sdk-packager-master/osx/data/script  
  inflating: kivy-sdk-packager-master/osx/package_app.py  
   creating: kivy-sdk-packager-master/win/
 extracting: kivy-sdk-packager-master/win/__init__.py  
  inflating: kivy-sdk-packager-master/win/common.py  
   creating: kivy-sdk-packager-master/win/data/
  inflating: kivy-sdk-packager-master/win/data/README.txt  
  inflating: kivy-sdk-packager-master/win/data/kivy-bash.bat  
  inflating: kivy-sdk-packager-master/win/data/kivy.bat  
  inflating: kivy-sdk-packager-master/win/data/kivybash.rc  
  inflating: kivy-sdk-packager-master/win/data/kivyenv.sh  
  inflating: kivy-sdk-packager-master/win/data/kivywineenv.sh  
   creating: kivy-sdk-packager-master/win/data/tools/
  inflating: kivy-sdk-packager-master/win/data/tools/cython.bat  
 extracting: kivy-sdk-packager-master/win/data/tools/easy_install.bat  
  inflating: kivy-sdk-packager-master/win/data/tools/link_msysgit.bat  
  inflating: kivy-sdk-packager-master/win/data/tools/make-portable-python.py  
 extracting: kivy-sdk-packager-master/win/data/tools/pip.bat  
  inflating: kivy-sdk-packager-master/win/data/tools/register-python.py  
  inflating: kivy-sdk-packager-master/win/dropbox.py  
  inflating: kivy-sdk-packager-master/win/gdrive.py  
  inflating: kivy-sdk-packager-master/win/glew.py  
  inflating: kivy-sdk-packager-master/win/gstreamer.py  
  inflating: kivy-sdk-packager-master/win/kivy_win_sdk.py  
  inflating: kivy-sdk-packager-master/win/readme  
  inflating: kivy-sdk-packager-master/win/sdl2.py  
# check if Kivy.app exists in local dir
# Kivy.app found at /Applications/Kivy.app
# copying it to platform dir
# Install platform
# Check application requirements
# Remove directory and subdirectory /Users/nico/packer/pack/.buildozer/applibs
# Create directory /Users/nico/packer/pack/.buildozer/applibs
# Run 'virtualenv --python=python2.7 ./venv'
# Cwd /Users/nico/packer/pack/.buildozer
New python executable in /Users/nico/packer/pack/.buildozer/venv/bin/python2.7
Also creating executable in /Users/nico/packer/pack/.buildozer/venv/bin/python
Installing setuptools, pip, wheel...
  Complete output from command /Users/nico/packer/p...r/venv/bin/python2.7 - setuptools pip wheel:
  Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
  File "//anaconda/lib/python2.7/tempfile.py", line 32, in <module>
    import io as _io
  File "//anaconda/lib/python2.7/io.py", line 51, in <module>
    import _io
ImportError: dlopen(/Users/nico/packer/pack/.buildozer/venv/lib/python2.7/lib-dynload/_io.so, 2): Symbol not found: __PyCodecInfo_GetIncrementalDecoder
  Referenced from: /Users/nico/packer/pack/.buildozer/venv/lib/python2.7/lib-dynload/_io.so
  Expected in: dynamic lookup

----------------------------------------
...Installing setuptools, pip, wheel...done.
Traceback (most recent call last):
  File "//anaconda/lib/python2.7/site-packages/virtualenv.py", line 2316, in <module>
    main()
  File "//anaconda/lib/python2.7/site-packages/virtualenv.py", line 708, in main
    symlink=options.symlink)
  File "//anaconda/lib/python2.7/site-packages/virtualenv.py", line 941, in create_environment
    download=download,
  File "//anaconda/lib/python2.7/site-packages/virtualenv.py", line 897, in install_wheel
    call_subprocess(cmd, show_stdout=False, extra_env=env, stdin=SCRIPT)
  File "//anaconda/lib/python2.7/site-packages/virtualenv.py", line 792, in call_subprocess
    % (cmd_desc, proc.returncode))
OSError: Command /Users/nico/packer/p...r/venv/bin/python2.7 - setuptools pip wheel failed with error code 1
Running virtualenv with interpreter //anaconda/bin/python2.7
# Command failed: virtualenv --python=python2.7 ./venv
# 
# Buildozer failed to execute the last command
# The error might be hidden in the log above this error
# Please read the full log, and search for it before
# raising an issue with buildozer itself.
# In case of a bug report, please add a full log with log_level = 2
```

Is suspect a problem with my virtual environment. I run on a mac with yosemite 10.10.5 and I am using python with anaconda.
BTW: log_level is set to 2 
BTW2: if I run the command again it will result in the follwing
```
mac:pack nico$ buildozer osx debug
# Check configuration tokens
# Ensure build layout
# Check configuration tokens
# Preparing build
# Check requirements for osx
# Check if kivy-sdk-packager exists
# kivy-sdk-packager found at /Users/nico/packer/pack/.buildozer/osx/platform
# check if Kivy.app exists in local dir
# Kivy.app found at /Users/nico/packer/pack/.buildozer/osx/platform/kivy-sdk-packager-master/osx
# Install platform
# Check application requirements
# Remove directory and subdirectory /Users/nico/packer/pack/.buildozer/applibs
# Create directory /Users/nico/packer/pack/.buildozer/applibs
# Run 'bash -c "source venv/bin/activate && env"'
# Cwd /Users/nico/packer/pack/.buildozer
bash: venv/bin/activate: No such file or directory
# Command failed: bash -c "source venv/bin/activate && env"
# 
# Buildozer failed to execute the last command
# The error might be hidden in the log above this error
# Please read the full log, and search for it before
# raising an issue with buildozer itself.
# In case of a bug report, please add a full log with log_level = 2
```

**Comment by lutznico** [comment]
This can easily be solved by downgrading to python 2.7.9 as can be read here https://github.com/conda/conda/issues/1367 and [here](http://chris35wills.github.io/conda_python_version/)
So essentially this has nothing to do with buildozer but with conda. Mark as closed please.

**Comment by kdheepak** [comment]
If anyone is still interested in this, I was able to solve this by using conda to install virtualenv before installing buildozer.
